cpp/JsiWorkletApi.cpp

```diff
@@ -45,18 +41,12 @@ JsiWorkletApi::createWorkletContext(const std::string &name) {
 
 void JsiWorkletApi::addDecorator(std::shared_ptr<JsiBaseDecorator> decorator) {
   // Decorate default context
-  JsiWorkletContext::addDecorator(decorator);
+  JsiWorkletContext::getDefaultInstance()->addDecorator(decorator);
 }
 
 std::shared_ptr<JsiWorkletApi> JsiWorkletApi::getInstance() {
   if (instance == nullptr) {
     instance = std::make_shared<JsiWorkletApi>();
-    // Add default decorators
-    JsiWorkletContext::addDecorator(
-        std::make_shared<JsiSetImmediateDecorator>());
-    JsiWorkletContext::addDecorator(
-        std::make_shared<JsiPerformanceDecorator>());
-    JsiWorkletContext::addDecorator(std::make_shared<JsiConsoleDecorator>());
```

Comment:
Should this be moved into the creation of the default context? Now it seems like no decorators are installed.

Reply:
Yeah the problem here is that we can't move this into the init call because the init call gets called twice (at least in my case). Maybe we need to refactor a bit here.